src/Standards/Generic/Sniffs/NamingConventions/CamelCapsFunctionNameSniff.php

```diff
@@ -113,7 +113,6 @@ protected function processTokenWithinScope(File $phpcsFile, $stackPtr, $currScop
 
         $methodName = $phpcsFile->getDeclarationName($stackPtr);
         if ($methodName === null) {
-            // Ignore closures.
```

Comment:
Historical context: there was a time in which PHPCS didn't retokenize the `T_FUNCTION` token to `T_CLOSURE` for closures, so at that time, the comment was perfectly correct.
The comment just wasn't updated when the tokenizer change was made.

```diff
+            // Live coding or parse error. Bow out.
```

(here and below)

Reply:
Done